ctapipe/containers.py

```diff
@@ -1143,3 +1214,81 @@ class ArrayEventContainer(Container):
         default_factory=MonitoringContainer,
         description="container for event-wise monitoring data (MON)",
     )
+
+
+class SchedulingBlockContainer(Container):
+    """Stores information about the scheduling block. This is a simplified version
+    of the SB model, only storing what is necessary for analysis. From [cta_sb_ob]_
+
+    References
+    ----------
+    .. [cta_sb_ob] Oya et al, Scheduling Block Data Model Specification,
+        CTA-SPE-COM-000000-0003, Issue 1, Rev. c
+    """
+
+    default_prefix = ""
+    sb_id = Field(UNKNOWN_ID, "Scheduling block ID", type=np.uint64)
+    sb_type = Field(
+        SchedulingBlockType.UNKNOWN,
+        description="Type of scheduling block",
+        type=SchedulingBlockType,
+    )
+    producer_id = Field(
+        "unknown",
+        "Origin of the sb_id, i.e. name of the telescope site or 'simulation'",
+        type=str,
+    )
+    observing_mode = Field(
+        ObservingMode.UNKNOWN,
+        "Defines how observations within the Scheduling Block are distributed in space",
+        type=ObservingMode,
+    )
+    pointing_mode = Field(
+        PointingMode.UNKNOWN, "Defines how the telescope drives move", type=PointingMode
+    )
+
+
+class ObservationBlockContainer(Container):
+    """Stores information about the observation"""
+
+    default_prefix = ""
+    obs_id = Field(UNKNOWN_ID, "Observation Block ID", type=np.uint64)
+    sb_id = Field(UNKNOWN_ID, "ID of the parent SchedulingBlock", type=np.uint64)
+    producer_id = Field(
+        "unknown",
+        "Origin of the obs_id, i.e. name of the telescope site or 'simulation'",
+        type=str,
+    )
+
+    state = Field(
+        ObservationBlockState.UNKNOWN, "State of this OB", type=ObservationBlockState
+    )
+
+    subarray_pointing_lat = Field(
+        nan * u.deg,
+        "latitude of the nominal center coordinate of this observation",
+        unit=u.deg,
+    )
+
+    subarray_pointing_lon = Field(
+        nan * u.deg,
+        "longitude of the nominal center coordinate of this observation",
+        unit=u.deg,
+    )
+
+    subarray_pointing_frame = Field(
+        CoordinateFrameType.UNKNOWN,
+        (
+            "Frame in which the subarray_target is non-moving. If the frame is altaz, "
+            "The meaning of lat,lon is altitude,azimuth.  If it is icrs, the meaning is "
+            "RA,Dec"
```

Comment:
sorry, one last comment, inconsitent order here:

```
            "The meaning of lat,lon is altitude,azimuth.  If it is icrs, the meaning is "
            "RA,Dec"
```

ra is lon, dec is lat.

Reply:
good catch